articles/quickstart.md

```diff
@@ -488,41 +479,11 @@ Init:One  0s=490  1s=510  agree=1000
 
 Our statistics for the first qubit haven't changed (50-50 chance of a 0 or a 1), but now when we measure the second qubit, it is __always__ the same as what we measured for the first qubit. Our `CNOT` has entangled the two qubits, so that whatever happens to one of them, happens to the other. If you reversed the measurements (did the second qubit before the first), the same thing would happen. The first measurement would be random and the second would be in lock step with whatever was discovered for the first.
 
+Congratulations, you've written your first quantum program!
 
-## Estimating Resources
```

Comment:
I totally agree with removing this from the current quickstart, but it'd be good if we could land it somewhere else instead if at all possible.

Reply:
Perhaps a future tutorial demonstrating the machines beyond the full state simulator? Would probably fit perfectly to a single tutorial if we just provide a bare bones situation of when each machine would be useful, and then quickly work through running on it and examining the output.

Reply:
Yes, I do agree with this, and we should definitely use the material in another tutorial or how to